packages/core/admin/admin/src/content-manager/pages/ListView/components/BulkActionButtons/SelectedEntriesModal/index.js

```diff
@@ -11,28 +11,111 @@ import {
   Tbody,
   Tr,
   Td,
+  IconButton,
+  Flex,
+  Icon,
 } from '@strapi/design-system';
-import { useTableContext, Table } from '@strapi/helper-plugin';
+import { useTableContext, Table, getYupInnerErrors } from '@strapi/helper-plugin';
+import { Pencil, CrossCircle, CheckCircle } from '@strapi/icons';
 import PropTypes from 'prop-types';
 import { useIntl } from 'react-intl';
 import { useSelector } from 'react-redux';
+import { Link, useHistory } from 'react-router-dom';
 
-import { getTrad } from '../../../../../utils';
+import { getTrad, createYupSchema } from '../../../../../utils';
 import { listViewDomain } from '../../../selectors';
 import { Body } from '../../Body';
 
+/* -------------------------------------------------------------------------------------------------
+ * EntryValidationText
+ * -----------------------------------------------------------------------------------------------*/
+
+const EntryValidationText = ({ errors, isPublished }) => {
+  const { formatMessage } = useIntl();
+
+  if (errors) {
+    return (
+      <Flex gap={2}>
+        <Icon color="danger600" as={CrossCircle} />
+        <Typography textColor="danger600" variant="omega" fontWeight="semiBold">
+          {Object.keys(errors)
+            .map((key) =>
+              formatMessage(
+                { id: `${errors[key].id}.withField`, defaultMessage: errors[key].defaultMessage },
+                { field: key }
+              )
+            )
+            .join(' ')}
```

Comment:
How come we can't use `useAPIErrors` hook here?

Reply:
I don't believe these are API errors. I think these are the Yup validation errors.

Reply:
In that case can we just use `Object.entries([key, error])`? :)